src/Utilities/BrowserConsoleLoggerTransport.ts

```diff
@@ -0,0 +1,43 @@
+import Transport from 'winston-transport';
+import _ from 'lodash';
+
+export default class BrowserConsoleLoggerTransport extends Transport {
```

Comment:
There are tests in Winston that validate API compliance, might be useful in the future for debugging.

https://github.com/winstonjs/abstract-winston-transport/tree/master